telemetry/aiwrapper.go

```diff
@@ -0,0 +1,65 @@
+package telemetry
+
+import (
+	"github.com/Azure/azure-container-networking/aitelemetry"
+	"github.com/Azure/azure-container-networking/log"
+)
+
+var (
+	aiMetadata     string
+	th             aitelemetry.TelemetryHandle
+	gDisableAll    bool
```

Comment:
what's the g prefix for?

Reply:
g for global..already using disableAll in CreateAITelemetryHandle..added this to prevent conflict